src/documentation_builder/lib/quattordocbuild/builder.py

```diff
@@ -0,0 +1,153 @@
+"""Build documentation from quattor sources."""
+
+import os
+import sys
+import jinja2
```

Comment:
it just works, see eg https://github.com/quattor/release-helper/pull/1/files#diff-4fe6142ecb6b8967d883339ec1384639R21